ScoutSuite/providers/gcp/resources/kms/keys.py

```diff
@@ -42,10 +42,13 @@ def _parse_key(self, raw_key):
         key_dict['purpose'] = raw_key['purpose']
         key_dict['rotation_period'] = raw_key.get('rotationPeriod', None)
         if key_dict['rotation_period']:
-            key_dict['rotation_period'] = int("".join(filter(str.isdigit, key_dict['rotation_period'])))
+            rotation_period = int("".join(filter(str.isdigit, key_dict['rotation_period'])))
+            # check if rotation period less than 90 days or 7776000 seconds
+            key_dict['rotation_period'] = rotation_period <= 7776000
 
         key_dict['next_rotation_time_days'] = None
         if key_dict['next_rotation_datetime']:
             next_rotation_time = dateutil.parser.parse(key_dict['next_rotation_datetime']) - datetime.now(timezone.utc)
-            key_dict['next_rotation_time_days'] = next_rotation_time.days
+            # check if next rotation period less than 90 days
+            key_dict['next_rotation_time_days'] = next_rotation_time.days <= 90
```

Comment:
How is this being displayed in the UI? You're moving the logic of the rule to the parsing but do these 2 variables still make sense considering what they are now storing?

Reply:
Fix it.